rpc/main.go

```diff
@@ -0,0 +1,274 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+	"unsafe"
+
+	"github.com/dgraph-io/badger/v3"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/gin-contrib/cors"
+	"github.com/gin-gonic/gin"
+	"github.com/go-logr/logr"
+	"github.com/go-logr/zerologr"
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog"
+	"github.com/stackup-wallet/stackup-bundler/pkg/altmempools"
+	"github.com/stackup-wallet/stackup-bundler/pkg/bundler"
+	"github.com/stackup-wallet/stackup-bundler/pkg/client"
+	"github.com/stackup-wallet/stackup-bundler/pkg/gas"
+	"github.com/stackup-wallet/stackup-bundler/pkg/jsonrpc"
+	"github.com/stackup-wallet/stackup-bundler/pkg/mempool"
+	"github.com/stackup-wallet/stackup-bundler/pkg/modules/batch"
+	"github.com/stackup-wallet/stackup-bundler/pkg/modules/checks"
+	"github.com/stackup-wallet/stackup-bundler/pkg/modules/entities"
+	"github.com/stackup-wallet/stackup-bundler/pkg/modules/expire"
+	"github.com/stackup-wallet/stackup-bundler/pkg/modules/gasprice"
+	"github.com/stackup-wallet/stackup-bundler/pkg/signer"
+	"go.opentelemetry.io/otel"
+
+	"github.com/blndgs/bundler/conf"
+	"github.com/blndgs/bundler/solution"
+	"github.com/blndgs/bundler/srv"
+	"github.com/blndgs/bundler/validations"
+)
+
+func main() {
+	values := conf.GetValues()
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
Maybe not needed because it takes hitting `ctrl-c' twice to exit.

Reply:
fixed this